realm/realm-library/src/androidTest/java/io/realm/CollectionTests.java

```diff
@@ -84,20 +84,22 @@ protected void populateRealm(Realm realm, int objects) {
         realm.beginTransaction();
         realm.delete(AllJavaTypes.class);
         realm.delete(NonLatinFieldNames.class);
-        for (int i = 0; i < objects; i++) {
-            AllJavaTypes obj = realm.createObject(AllJavaTypes.class, i);
-            fillObject(i, objects, obj);
-            NonLatinFieldNames nonLatinFieldNames = realm.createObject(NonLatinFieldNames.class);
-            nonLatinFieldNames.set델타(i);
-            nonLatinFieldNames.setΔέλτα(i);
-        }
-
-        // Add all items to the RealmList on the first object
-        AllJavaTypes firstObj = realm.where(AllJavaTypes.class).equalTo(AllJavaTypes.FIELD_LONG, 0).findFirst();
-        RealmResults<AllJavaTypes> listData = realm.where(AllJavaTypes.class).findAllSorted(AllJavaTypes.FIELD_LONG, Sort.ASCENDING);
-        RealmList<AllJavaTypes> list = firstObj.getFieldList();
-        for (int i = 0; i < listData.size(); i++) {
-            list.add(listData.get(i));
+        if (objects > 0) {
```

Comment:
m is 0, The reason i added that if statement is because of the code below the loop: https://github.com/realm/realm-java/pull/2124/files#diff-af07352af4c866c231b76d8b9a522827R96 It would blow up for the special case where `objects = 0`